0001_initial_schema.up.sql

```diff
@@ -0,0 +1,27 @@
+
+BEGIN
+
+    CREATE TABLE Clothes (
+        id SERIAL PRIMARY KEY,
+        asset_url VARCHAR ( 50 ) UNIQUE NOT NULL,
```

Comment:
Do we need a constraint of 50 characters here? considering that sometimes urls are long.